src/main/resources/transformation/metafix-wip.fix

```diff
@@ -0,0 +1,499 @@
+      </data>
+																								*/
+	do list(path:"$i.d", "var":"$j")
+		unless exists("RSWK[].$last.subjectDate")
+			replace_all("$j","\\s?[:.]?$","")
+			copy_field("$j", "RSWK[].$last.subjectDate")
+		end
+	end
+/*
+			      <data name="subjectConference" source="611??.a">
+        <occurrence only="1" sameEntity="true" />
+        <replace pattern="\.?$" with="" />
+      </data>
+																								*/
+	do list(path:"$i.a", "var":"$j")
+		unless exists("RSWK[].$last.subjectConference")
+			replace_all("$j","\\.?$","")
+			copy_field("$j", "RSWK[].$last.subjectConference")
+		end
+	end
+/*
+      <data name="subjectChronological" source="6[105]0??.y|6[15]1??.y||648??.a">
+        <occurrence only="1" sameEntity="true" />
+        <replace pattern="\.?$" with="" />
+      </data>
+																											*/
+	do list(path:"$i.y", "var":"$j")
+		unless exists("RSWK[].$last.subjectChronological")
+			replace_all("$j","\\.?$","")
+			copy_field("$j", "RSWK[].$last.subjectChronological")
+		end
+	end
+/*
+			      <data name="subjectGeoName" source="6[105]0??.z|6[15]1??.z|651??.a">
+        <occurrence only="1" sameEntity="true" />
+        <replace pattern="\.?$" with="" />
+        <unique />
+      </data>																											*/
+	do list(path:"$i.z", "var":"$j")
+		unless exists("RSWK[].$last.subjectGeoName")
+			replace_all("$j","\\.?$","")
+			copy_field("$j", "RSWK[].$last.subjectGeoName")
+		end
+	end
+
+	remove_field("RSWK[].$last.control_611")
+end
+/*
+
+648*/
+do list(path:"648??", "var":"$i")
+	add_field("RSWK[].$append.control_648", "true")
+	/*
+      <entity name="subjectIdentifier[]" flushWith="600??|6[15][01]??|648??" sameEntity="true">
+        <data name="" source="600??.0|6[15][01]??.0|648??.0" />
+      </entity>
+      <entity name="identifierGND[]" flushWith="600??|6[15][01]??|648??" sameEntity="true">
+        <data name="" source="600??.0|6[15][01]??.0|648??.0">
+          <regexp match="(?&lt;=\(DE-588\)).*$" />
+        </data>*/
+	do list(path:"$i.0", "var":"$j")
+		unless exists("RSWK[].$last.subjectIdentifier[]")
+			set_array("RSWK[].$last.subjectIdentifier[]")
+		end
+		copy_field("$j", "RSWK[].$last.subjectIdentifier[].$append")
+		if any_match("$j", "^\\(DE-588\\).*")
+			replace_all("$j", "^\\(DE-588\\)", "")
+			unless exists("RSWK[].$last.identifierGND[]")
+				set_array("RSWK[].$last.identifierGND[]")
+			end
+			copy_field("$j", "RSWK[].$last.identifierGND[].$append")
+		end
+	end
+/*		
+				      <data name="subjectChronological" source="6[105]0??.y|6[15]1??.y||648??.a">
+        <occurrence only="1" sameEntity="true" />
+        <replace pattern="\.?$" with="" />
+      </data>*/
+	do list(path:"$i.a", "var":"$j")
+		unless exists("RSWK[].$last.subjectChronological")
+			replace_all("$j","\\.?$","")
+			copy_field("$j", "RSWK[].$last.subjectChronological")
+		end
+	end
+	remove_field("RSWK[].$last.control_648")
+end
+/*
+
+650*/
+do list(path:"650??", "var":"$i")
```

Comment:
This should only interate over one object of 650.
But that does not fit to the provided data:
See other comments.

Reply:
https://github.com/hbz/limetrans/blob/ac36a22c961e6e5f03189ca3e769814fe2921e32/src/test/resources/limetrans/input/alma.xml#L12980-L12995

https://github.com/hbz/limetrans/blob/6cd42478ed1264f1fd8866b7a69d5bfa6daff093/src/test/resources/limetrans/input/mf_alma.json#L13163-L13180

Reply:
Yes, that's the same problem as in my [previous comment](https://github.com/hbz/limetrans/pull/79#discussion_r794657316): The repeated fields `650 0` are turned into an array which is then passed as a single item to `do list()`.

Input:

```xml
<datafield tag="650" ind1=" " ind2="0">
  <subfield code="a">Management science</subfield>
</datafield>
<datafield tag="650" ind1=" " ind2="0">
  <subfield code="a">Industrial management&#x2014;Environmental aspects</subfield>
</datafield>
<datafield tag="650" ind1=" " ind2="0">
  <subfield code="a">Business ethics</subfield>
</datafield>
<datafield tag="650" ind1=" " ind2="0">
  <subfield code="a">Environmental economics</subfield>
</datafield>
<datafield tag="650" ind1=" " ind2="0">
  <subfield code="a">Business</subfield>
</datafield>
```

Reply:
Reported in metafacture/metafacture-fix#119.